packages/sdk-js/src/index.ts

```diff
@@ -5,23 +5,11 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import * as core from '@kiltprotocol/core'
-import Message, * as Messaging from '@kiltprotocol/messaging'
-import { BlockchainUtils } from '@kiltprotocol/chain-helpers'
-import * as ChainHelpers from '@kiltprotocol/chain-helpers'
-import * as Did from '@kiltprotocol/did'
-import * as Utils from '@kiltprotocol/utils'
-
 export * from '@kiltprotocol/core'
+export { Message } from '@kiltprotocol/messaging'
+export * as Messaging from '@kiltprotocol/messaging'
+export { BlockchainUtils } from '@kiltprotocol/chain-helpers'
+export * as ChainHelpers from '@kiltprotocol/chain-helpers'
+export * as Did from '@kiltprotocol/did'
+export * as Utils from '@kiltprotocol/utils'
 export * from '@kiltprotocol/types'
-export { Message, Messaging, BlockchainUtils, ChainHelpers, Utils, Did }
-
-export default {
```

Comment:
C:
Thinking about leaving a default export here, just to have a nicer import
`import Kilt from '@kiltprotocol/sdk-js`.
I can also think of just exporting everything in the kilt folder in `@kiltprotocol/core` as `Kilt`, so we could do
```
import { Kilt } from '@kiltprotocol/sdk-js'
Kilt.init({})
```
What do you think?

Reply:
Exporting `Kilt` as Kilt would mean that someone using the rest of `@kiltprotocol/core` would have to import with something like `import * as Kilt from `@kiltprotocol/core`, and then having a `Kilt.Kilt` pacakge when using Kilt inside core?

Reply:
And I would not mix in a default export again, cause it might be quite confusing.

Reply:
yes, `import * as Kilt` would not work well then, but I would also not advertise that, ~since tree-shaking is way harder this way for bundlers~ <-- Seems I am wrong and it IS working: https://github.com/webpack/webpack/issues/2713.
I agree we shouldn't have a default export here.

Reply:
Ok, so I don't feel super comfortable with this, but a lot of other projects are also using this way of importing and tree-shaking seems to work, so you are good to go